src/main/java/edu/ucsb/cs156/team02/controllers/UCSBSubjectController.java

```diff
@@ -0,0 +1,259 @@
+package edu.ucsb.cs156.team02.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.ucsb.cs156.team02.entities.Todo;
+import edu.ucsb.cs156.team02.entities.UCSBSubject;
+import edu.ucsb.cs156.team02.models.CurrentUser;
+import edu.ucsb.cs156.team02.repositories.UCSBSubjectRepository;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+@Api(description = "UCSBSubjects")
+@RequestMapping("/api/UCSBSubjects/")
+@RestController
+@Slf4j
+public class UCSBSubjectController extends ApiController {
+
+//    /**
```

Comment:
Perhaps we could delete the Todo code?

Reply:
I think for the `main` branch it is probably smart to delete the `Todo` code.